src/config.c

```diff
@@ -537,6 +546,7 @@ int config_print(struct re_printf *pf, const struct config *cfg)
 			 "\n"
 			 "# Network\n"
 			 "net_interface\t\t%s\n"
+			 "net_af\t\t%s\n"
```

Comment:
The corresponding `cfg->net.af` to string conversion is missing here.

Reply:
Fixed by https://github.com/baresip/baresip/pull/2490/commits/de4cd581857341d38f412fda57f6ba1aedccca3f